pkg/ami/api.go

```diff
@@ -54,6 +54,9 @@ func Use(ec2api ec2iface.EC2API, ng *api.NodeGroupBase) error {
 	case "ebs":
 		if !api.IsSetAndNonEmptyString(ng.VolumeName) {
 			ng.VolumeName = image.RootDeviceName
+			if ng.AMIFamily == api.NodeImageFamilyBottlerocket {
+				ng.VolumeName = aws.String("/dev/xvdb")
```

Comment:
Maybe better as a constant `BottlerocketDataDisk` or something? (So the name explains why we're setting it)